src/common/options/global.yaml.in

```diff
@@ -175,11 +173,7 @@ options:
   type: str
   level: advanced
   desc: monitor(s) to use overriding the MonMap
-  fmt_desc: the list of monitors for the cluster to
-    **initially** contact when beginning a new instance of communication with the
-    Ceph cluster.  This overrides the known monitor list derived from MonMap
-    updates sent to older Ceph instances (like librados cluster handles).  It is
-    expected this option is primarily useful for debugging.
+  fmt_desc: This is the list of monitors that the cluster **initially** contacts when first establishing communication with the Ceph cluster. This overrides the known monitor list that is derived from MonMap updates sent to older Ceph instances (like librados cluster handles). This option is expected to be useful primarily for debugging.
```

Comment:
first sentence: the cluster isn't necessarily the user of this option, it'd be more likely to be a client.

Maybe something like 'ceph process' - it could be a daemon that is part of the cluster or a client.